prac_07/guitars.py

```diff
@@ -0,0 +1,24 @@
+from prac_07.guitar_storage import Guitar
+
+print("My guitars!")
+name = "j"
+guitars = []
+while name != "":
+    name = input("Name: ")
+    if name != "":
```

Comment:
See the problem with this? You've already got a while loop asking the same question?
Please read the chapter 2 notes again.